fast_node.go

```diff
@@ -11,7 +12,7 @@ type FastNode struct {
 	key                  []byte
```

Comment:
wdyt about keeping key? Think we should keep it or remove it to help reduce the committed data overhead?

Reply:
I removed the key from being written as a value of the fast node on the disk. We still need to keep it as a member of the struct for retrieval from disk and managing it in memory

Reply:
oh awesome, great trade-off taken!